### PR TITLE
Fix issue in BitLink component style on demo pages (#486)

### LINKED ISSUE #486: Incorrect style for BitLink component in demo pages
BitLink component not showing in BitLink demo page and component page because of header link styles.

Page :
link

Code :
code

### PR DESCRIPTION
**Pull request by SomayeEbrahimi**
This closes #486

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/styles/styles.scss [hunk 1]
```diff
@@ -45,7 +45,7 @@ p {
     }
 }
 
-.bit-lnk-fluent > a {
+.navbar > .bit-lnk-fluent > a {
     float: left;
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/styles/styles.scss:49]: why float? I'm not comfortable with it.
and there is another float attribute in this selector, please investigate and improve it.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/styles/styles.scss (hunk 1 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/styles/styles.scss:49]: I meant the "float" attribute, not the selector's name.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor [hunk 2]
```diff
@@ -3,11 +3,12 @@
 
 <div>
     <div class="navbar">
-        <BitLink Href="#">Home</BitLink>
-        <BitLink Href="/component/button" Class="active">Component</BitLink>
-        <BitLink Href="#">Description</BitLink>
-
         <h3 class="header-title">BIT Component</h3>
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor:6]: "bit" is not an abbr. and should not be typed in all CAPS.

Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor:6]: and also component here should be plural (components)

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor [hunk 3]
```diff
@@ -3,11 +3,12 @@
...
+        <div class="links">
+            <BitLink Href="#">Home</BitLink>
+            <BitLink Href="/component/button" Class="active">Component</BitLink>
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor:9]: change all the routes and labels related to component into the plural form: "components"

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor (hunk 3 above)
Comment (SomayeEbrahimi) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ComponentLayout.razor:9]: This has been done in another branch and was resolved after merging with develop branch

**Comment by msynk** [comment]
I'm closing this PR because of the unrelated commits in it

**Pull request closed without merging**